GMG/Core/Models/ScoreFactory/ScoreFactory.swift

```diff
@@ -117,13 +119,31 @@ final class ScoreFactory {
 
         let swiftF0Notes: [SwiftF0.Note] = NoteConverter.convert(pitchResults)
         let notes: [Note] = swiftF0Notes.map { note in
-            return convertSwiftF0NoteToNote(note)
+            return note.note
         }
 
         return notes
     }
 
-    private func convertSwiftF0NoteToNote(_ swiftF0Note: SwiftF0.Note) -> Note {
+    private func mergeChordCells(_ chordCells: [ChordCell]) -> [ChordCell] {
```

Comment:
함수 이름에서, '연속된' 이라는 뜻이 들어가면 보다 더 쉽게 코드를 이해할 수 있을 것 같습니다


```suggestion
    private func mergeConsecutiveChordCells(_ chordCells: [ChordCell]) -> [ChordCell] {
```